### ISSUE TITLE
ClassNotFoundException with IntelliJ 192

### ISSUE THREAD

**Issue by ice1000** [JetBrains/Grammar-Kit#214]
I got this exception:

```
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8
java.lang.NoClassDefFoundError: com/intellij/ui/JBColor
        at com.intellij.util.CachedValueLeakChecker.checkProvider(CachedValueLeakChecker.java:51)
        at com.intellij.util.CachedValuesManagerImpl.freshCachedValue(CachedValuesManagerImpl.java:94)
        at com.intellij.util.CachedValuesManagerImpl.getCachedValueFromExHolder(CachedValuesManagerImpl.java:70)
        at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:45)
        at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:147)
        at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:129)
        at org.intellij.grammar.generator.RuleGraphHelper.getTokenTextToNameMap(RuleGraphHelper.java:209)
        at org.intellij.grammar.generator.ParserGenerator.<init>(ParserGenerator.java:194)
        at org.intellij.grammar.Main.main(Main.java:85)
Caused by: java.lang.ClassNotFoundException: com.intellij.ui.JBColor
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
        ... 9 more
```

I'm using IC-192.4488.21

**Comment by ice1000** [comment]
Maybe I should migrate this issue to the gradle plugin 🤔

**Comment by hurricup** [comment]
Fixed in plugin 2019.2

**Comment by ice1000** [comment]
Thanks! Closing.
